sale_multi_operating_unit/views/sale_order_view.xml

```diff
@@ -0,0 +1,68 @@
+<odoo>
+    <!-- Copyright (C) 2019 Open Source Integrators
+    Copyright (C) 2019 Serpent Consulting Services
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+    <record id="extend_view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[@name='other_information']" position="after">
+                <page string="Internal Quote Requests">
+                    <field name="quote_ids" context="{'active_id': active_id}">
```

Comment:
@Dangar-Serpentcs @max3903 We need this line to be changed to:
`<field name="quote_ids" context="{'active_id': id}">`

The active_id being passed in could be coming from a crm.lead if you got from Lead > Quotations Button > Quotation > Internal Quotation. I believe this could be the case for other models that have links to Sales Orders so I believe it is best to store active_id with the ID of the SO we are on instead of the the active_id currently in the context.

Reply:
Improved as par your comment.